com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitGUI.cs

```diff
@@ -254,18 +254,12 @@ public static void SetupBaseUnlitKeywords(this Material material)
             }
 
             // DoubleSidedGI has to be synced with our double sided toggle
-            var serializedObject = new SerializedObject(material);
-            bool doubleSidedGI = false;
             if (doubleSidedGIMode == DoubleSidedGIMode.Auto)
-                doubleSidedGI = doubleSidedEnable;
+                material.doubleSidedGI = doubleSidedEnable;
             else if (doubleSidedGIMode == DoubleSidedGIMode.On)
-                doubleSidedGI = true;
+                material.doubleSidedGI = true;
             else if (doubleSidedGIMode == DoubleSidedGIMode.Off)
-                doubleSidedGI = false;
-            // material always call setdirty, so set only if new value is different
```

Comment:
Sanity check you have fixed that on C++ side?

Reply:
yep